docker-compose.ci.yml

```diff
@@ -27,3 +30,5 @@ services:
       - .:/obs
     working_dir: /obs
     command: make -C src/backend test
+  selenium:
```

Comment:
Why do we need this on travis? Why not use the local chromedriver?

Reply:
there is no local chromedriver in the 42.3 containers. I thought we can do this step by step